settings.gradle

```diff
@@ -47,7 +47,7 @@ if ("true".equals(System.getenv("H2O_BUILD_GRPC"))) {
 apply plugin: 'scala-cross-build'
 
 scalaCrossCompile {
-  targetVersions "2.10.4", "2.11.7"
+  targetVersions "2.10.7", "2.11.12"
```

Comment:
how is it related to Java 8 support? https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html
Just surprised that a simple upgrade in scala patch version is necessary to be compatible with Java 8.

Is it to be actually compatible with Java 11?

Reply:
It is not related to Java 8 but it is related to the other changes in this PR - our Jenkins builds uses Java 8 and everything works fine but trying to compile on Java 11/12 without this change produces an error without the patch:

Java HotSpot(TM) 64-Bit Server VM warning: Ignoring option MaxPermSize; support was removed in 8.0
error: error while loading package, Missing dependency 'object java.lang.Object in compiler mirror', required by /Users/mkurka/.gradle/caches/modules-2/files-2.1/org.scala-lang/scala-library/2.10.4/9aae4cb1802537d604e03688cab744ff47b31a7d/scala-library-2.10.4.jar(scala/package.class)
error: error while loading package, Missing dependency 'object java.lang.Object in compiler mirror', required by /Users/mkurka/.gradle/caches/modules-2/files-2.1/org.scala-lang/scala-library/2.10.4/9aae4cb1802537d604e03688cab744ff47b31a7d/scala-library-2.10.4.jar(scala/runtime/package.class)
error: scala.reflect.internal.MissingRequirementError: object java.lang.Object in compiler mirror not found.
        at scala.reflect.internal.MissingRequirementError$.signal(MissingRequirementError.scala:16)
        at scala.reflect.internal.MissingRequirementError$.notFound(MissingRequirementError.scala:17)
        at scala.reflect.internal.Mirrors$RootsBase.getModuleOrClass(Mirrors.scala:48)
        at scala.reflect.internal.Mirrors$RootsBase.getModuleOrClass(Mirrors.scala:40)

Reply:
(also added this info to the commit message)